src/Core/AndroidNative/maui/src/main/java/com/microsoft/maui/glide/MauiCustomViewTarget.java

```diff
@@ -46,11 +46,13 @@ public void onResourceReady(@NonNull Drawable resource, @Nullable Transition<? s
             return;
         this.completed = true;
 
-        // set the image
-        this.view.setImageDrawable(resource);
+        post(() -> {
+            // set the image
+            this.view.setImageDrawable(resource);
```

Comment:
The new aar file will need to be generated

Reply:
It was

Reply:
aw cool! I missed that when reviewing last.